src/wizards/generateProject/generationWizard.ts

```diff
@@ -202,6 +203,11 @@ export async function generateProjectWizard() {
   state.targetDir = await getTargetDirectory(state.artifactId);
 
   const projectGenState: ProjectGenState = state as ProjectGenState;
+  await sendTelemetry(VSCodeCommands.CREATE_PROJECT, {
```

Comment:
now 2 events are sent with this name, because an event is already sent on every command executed

Reply:
This should be prevented by https://github.com/redhat-developer/vscode-quarkus/pull/437/files#diff-7aedfdde409c92096e09505c33a32ab6c61b36645e58da8a159ca63144f9c364R26 . When set to `true` (as done for `generateProjectWizard` registration), the success is not reported as it's assumed the command will handle it.

Although I just noticed we don't set `status`. That should definitely be fixed.

Although I realized we don't set success